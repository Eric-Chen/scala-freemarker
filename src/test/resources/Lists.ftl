[#ftl]
<ul>
[#list list as l]
  <li>${l}</li>
[/#list]
</ul>

[#list range as r]${r}[#if r_has_next], [/#if][/#list]