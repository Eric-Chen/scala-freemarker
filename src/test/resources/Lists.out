<ul>
  <li>one</li>
  <li>two</li>
  <li>three</li>
</ul>

0, 1, 2, 3, 4, 5, 6, 7, 8, 9